packages/nectar/src/sagas/errors/handleErrors/handleErrors.saga.test.ts

```diff
@@ -77,26 +80,30 @@ describe('handle errors', () => {
           registrarAddress: community.registrarUrl
         })
       )
-      .put({ type: 'id.registrar.error' })
       .run()
   })
-  test('general error', async () => {
-    await expectSaga(
-      handleErrorsSaga,
-      errorsActions.addError({
-        type: 'sockets',
-        message: 'Registering username failed.',
-        communityId: 'general',
-        code: 500
-      })
-    )
-      .withReducer(
-        combineReducers({
-          [StoreKeys.Communities]: communitiesReducer,
-          [StoreKeys.Identity]: identityReducer
-        })
-      )
-      .put({ type: 'general.sockets.error' })
-      .run()
+
+  test('registrar validation error does not trigger re-registration', async () => {
+    const addErrorAction = errorsActions.addError({
+      type: SocketActionTypes.REGISTRAR,
+      message: ErrorMessages.INVALID_USERNAME,
+      communityId: community.id,
+      code: ErrorCodes.VALIDATION
+    })
+    testSaga(handleErrorsSaga, addErrorAction)
+      .next()
+      .isDone()
```

Comment:
How exactly do we assert anything here?

Reply:
The flow (and that's what is interesting for me in this case). 
It checks that the saga is finished and does not call anything else e.g doesn't put another action.